std/experimental/allocator/building_blocks/free_list.d

```diff
@@ -1045,6 +1047,28 @@ struct SharedFreeList(ParentAllocator,
         resetNodes();
         return result;
     }
+
+    /**
+    Nonstandard function that minimizes the memory usage of the freelist by
+    freeing each element in turn. Defined only if $(D ParentAllocator) defines
+    $(D deallocate).
+    */
+    static if (hasMember!(ParentAllocator, "deallocate") && !unchecked)
```

Comment:
Ah, of course. The constraint `!unchecked` is necessary because otherwise the function has no idea what size each block is...